extensions/wrapper/wrapper-common-mappers/src/main/java/de/sovity/edc/ext/wrapper/api/common/mappers/utils/UiAssetMapper.java

```diff
@@ -236,14 +264,18 @@ private JsonObjectBuilder getAssetProperties(
     private JsonObjectBuilder getAssetPrivateProperties(UiAssetCreateRequest uiAssetCreateRequest) {
         var privateProperties = Json.createObjectBuilder();
 
-        var stringProperties = uiAssetCreateRequest.getPrivateProperties();
```

Comment:
the `getPrivateProperties` function can be removed as well

Reply:
`getStringProperties` and `getJsonProperties` are also unused

Reply:
Cleaned up.